src/components/component.ts

```diff
@@ -356,15 +360,18 @@ export class Component {
 
   private _generateClipPath() {
     // The clip path will prevent content from overflowing its Component space.
-    // HACKHACK: IE <=9 does not respect the HTML base element in SVG.
+    this._clipPathID = Utils.DOM.generateUniqueClipPathId();
+    let clipPathParent = this._boxContainer.append("clipPath").attr("id", this._clipPathID);
+    this._addBox("clip-rect", clipPathParent);
+    this._updateClipPath();
+  }
+
+  private _updateClipPath() {
```

Comment:
I only see _updateClipPath() called by renderImmediately(). Are there other spots that should be calling it, or is that the only function that renders things?